### PR TITLE
Birthday flavor text

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -350,6 +353,21 @@ def flavor_text
     end
   end
 
+  def birthday_flavor_text
+    return nil unless current_user.birthday.present? &&
+                      current_user.birthday.to_date == Date.today &&
+                      rand(100) <= 10
```
Comment (cjdenio) [inline app/helpers/static_pages_helper.rb]: Could this be given a higher chance of happening? I'm interpreting this as a 1/10th chance of seeing a birthday message

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -350,6 +353,21 @@ def flavor_text
     end
   end
 
+  def birthday_flavor_text
+    return nil unless current_user.birthday.present? &&
```
Comment (cjdenio) [inline app/helpers/static_pages_helper.rb]: the `user_birthday?` helper also exists

File: app/helpers/static_pages_helper.rb (hunk 1 above)
Comment (maxwofford) [reply app/helpers/static_pages_helper.rb]: increasing!

File: app/helpers/static_pages_helper.rb (hunk 2 above)
Comment (maxwofford) [reply app/helpers/static_pages_helper.rb]: Refactored!

Comment (maxwofford) [reply app/helpers/static_pages_helper.rb]: What's that provided by? I can't find a `def user_birthday?`

**Review by cjdenio — APPROVED** [review]
Thanks!

File: app/helpers/static_pages_helper.rb (hunk 2 above)
Comment (cjdenio) [reply app/helpers/static_pages_helper.rb]: It's in `users_helper.rb`, on the `cjdenio/party-mode` branch

**Pull request merged**